src/frontend/komponenter/Felleskomponenter/Skjemasteg/Skjemasteg.tsx

```diff
@@ -19,6 +20,24 @@ interface IProps {
     skalViseNesteKnapp?: boolean;
 }
 
+const Container = styled.div`
+    padding: 2rem;
+
+    .typo-innholdstittel {
```

Comment:
Lage en StyledInnholdstittel av denne? :) Hadde vært nice å unngå klassenavn så langt det lar seg gjøre 🙌

Reply:
Kan godt det. Det blir fort veldig mange styled const'er i samme fil - er det et problem?

Reply:
Nei, dette er nok også smak og behag. Jeg synes det er en del enklere å holde oversikt over hvor denne stylen treffer når det er en egen styled component i stedet for å kalle klassenavn inne i styled components. Da slipper man å lete etter hvor denne klassen er lagt på en komponent.